backend/inspirehep/search/mappings/v7/records/hep.json

```diff
@@ -351,6 +351,153 @@
         },
         "type": "nested"
       },
+      "first_author": {
+        "properties": {
+          "suggest": {
+            "type": "completion"
+          },
+          "affiliations": {
+            "properties": {
+              "curated_relation": {
+                "type": "boolean"
+              },
+              "recid": {
+                "type": "integer"
+              },
+              "record": {
```

Comment:
do we care about all this fields? Don't we just need ``full_name``, ``first_name``, ``last_name`` and ``ids``?

Reply:
I thought that in the future we would want to query for more things for ```first_author```